tests/suites/test_suite_pkparse.data

```diff
@@ -103,8 +103,19 @@ depends_on:MBEDTLS_DES_C:MBEDTLS_SHA1_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_PKCS5_C:MBED
 pk_parse_keyfile_rsa:"data_files/pkcs8_pbes2_pbkdf2_des.key":"PolarSSLTest":0
 
 Parse Public RSA Key #1 (PKCS#8 wrapped)
-depends_on:MBEDTLS_MD5_C:MBEDTLS_PEM_PARSE_C
-pk_parse_public_keyfile_rsa:"data_files/format_gen.pub":0
+depends_on:MBEDTLS_PEM_PARSE_C
+pk_parse_public_keyfile_rsa:"data_files/rsa_pkcs8_2048_public.pem":0
+
+Parse Public RSA Key #1 (PKCS#8 wrapped, DER)
+pk_parse_public_keyfile_rsa:"data_files/rsa_pkcs8_2048_public.der":0
```

Comment:
The dependency on `MBEDTLS_RSA_C` seems to be missing here.

Reply:
no need, because [`pk_parse_public_keyfile_rsa`](https://github.com/ARMmbed/mbedtls/blob/development/tests/suites/test_suite_pkparse.function#L12) tests case is dependent on `MBEDTLS_RSA_C`